features/events/traps/pom.xml

```diff
@@ -76,6 +76,11 @@
     </plugins>
   </build>
   <dependencies>
+   <dependency>
+        <groupId>javax.ws.rs</groupId>
+        <artifactId>javax.ws.rs-api</artifactId>
+        <version>2.0.1</version>
```

Comment:
We already define the `javax.ws.rs-api` version in the top-level `pom.xml` file, so this line can be removed (to avoid future version conflicts).